### DISCUSSION TITLE
How do I include a third party .jar file in my build?

### DISCUSSION THREAD

**Discussion by webnoob** [ionic-team/capacitor#4593] [Q&A] [answered]
I'm trying to use this plugin: https://github.com/edu-com/cordova-plugin-amazon-login and it's instructions say:

> Copy login-with-amazon.jar file to platforms/android/libs/login-with-amazon.jar of your Cordova project.

What's the path I need so it will be included in the build in capacitor as this path is for cordova?

If I do:

1. Go to this folder
2. Create lobs
3. Add my jar file then right click Import as Lib

![image](https://user-images.githubusercontent.com/10404747/118954217-7585e500-b955-11eb-93f2-d9f45c146bfb.png)

Then it works in my app. I'd like to know how this can be done automatically though:

**Comment by webnoob** [comment, marked as answer]
For now, I've added a copy file routine to my build pipeline to handle this.

**Reply by thomasvidas** [reply]
That seems like the correct way to handle it, there isn't a way to automatically detect and include `.jar` or `.xcframework` libraries in builds at this time.

**Reply by webnoob** [reply]
Ah Ok - good to know.

It seems that Cordova handled this part so wasn't clear it was a manual process with Capacitor.
